README.md

```diff
@@ -1,4 +1,4 @@
-ACL (Access Control List)
+acl
```

Comment:
The explanation of ACEs moved down to the top of the Usage section (line 51). The Description section is supposed to stay pretty high-level, focusing on the question "what does it do?". This info seemed more pertinent to "how do I use it?".